packages/react/src/utils/TypeUtils.ts

```diff
@@ -0,0 +1 @@
+export type Override<T1, T2> = Omit<T1, keyof T2> & T2;
```

Comment:
I think if you do it that way the order in the props table will be correct
```suggestion
export type Override<T1, T2> = T2 & Omit<T1, keyof T2>;
```

Reply:
It does. 👍  Well spotted!